CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # spring-service
 
+## 3.3.1
+
+- RabbitMQ ssl mode will be configured based on parameter in external secret
```

Comment:
Nit: The official lingo would be "TLS support" (https://www.rabbitmq.com/ssl.html), I'd say all that config is more like an implementation detail

Reply:
Good catch, but I would stick to `ssl` as we have currently only spring services that use the `ssl` term and it would then be a little weird if there is sometimes "ssl" and sometimes "tls" mentioned.

Reply:
One could argue that TLS is the proper term altogether, but I don't want to get into bikeshedding here